packages/vapor/scss/components/blankslate.scss

```diff
@@ -1,6 +1,8 @@
 .blankslate {
+    --color: var(--title-text-color);
```

Comment:
I'm wondering what'S the benefit to declare a var for `--color ` instead of using the `--title-text-color` directly in the classes? :eyes:

Reply:
If I understand correctly, the new pattern is to use fewer variables and keep them as simple as possible. For instance:
.class {
    --color: var(--title-text-color);
    color: var(--color);
}
Then we could override --color inside the scope. By using --color, we can avoid too much repetition, --title-text-color was defined as variable because it's a default value and it will be reused a lot.

Reply:
To add on what Rosalie said, we should define variables when they are overriden in a `mod`. This way, the mod only has to redefine the variable instead of having to re-declare the color.

It doesn't seem like much but if your component has some nested components then the new variable will impact all the children